src/fairchem/core/models/base.py

```diff
@@ -272,3 +287,9 @@ def forward(self, data: Batch):
             out.update(self.output_heads[k](data, emb))
 
         return out
+
+    def get_backbone(self) -> BackboneInterface:
+        return self.backbone
+
+    def get_heads(self) -> dict[str, HeadInterface]:
+        return self.output_heads
```

Comment:
super nit, if these intended to serve as getter functions can we change them to @property and have the actual attributes be private `self._backbone`.